src/Schema/Tables/UsersData.sql

```diff
@@ -0,0 +1,16 @@
+﻿CREATE TABLE [dbo].[UsersData] 
+  ( 
+     [UserId]      INT NOT NULL IDENTITY(1, 1), 
+     [FirstName]   NVARCHAR(100) NOT NULL, 
+     [LastName]    NVARCHAR(100) NOT NULL, 
+     [BirthDate]   DATE NOT NULL, 
+     [PhoneNumber] VARCHAR(12) NOT NULL, 
```

Comment:
About PhoneNumber I left a comment to @FSuhov as well. Please comparate regarding this question.
Besides -  +380636576768 - 13 diget